lexicons/com/atproto/identity/sendPlcOp.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.sendPlcOp",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Validates a PLC operation to ensure that it doesn't violate a service's constraints or get the identity into a bad state, then submits it to the PLC registry",
```

Comment:
should we allow a param to skip validation, like we do for lexicons?

I'd be fine not making the validation optional to start, but maybe add it in the future.

Reply:
In my view, if you want to do that, you should just send the operation to PLC yourself. But yeah we can always add it later if we decide we want it 👍

Reply:
yes! good call